R/split_train_test.R

```diff
@@ -23,7 +32,42 @@ split_train_test <- function(d, outcome, percent_train = .8, seed) {
     stop(outcome_chr, " isn't a column in d.")
   if (!missing(seed))
     set.seed(seed)
-  train_rows <- caret::createDataPartition(dplyr::pull(d, !!outcome),
-                                           p = percent_train)[[1]]
-  list(train = d[train_rows, ], test = d[-train_rows, ])
+  if (!missing(grouping_col)){
```

Comment:
If you're going to have an `if` and an `else`, avoid the double negative of `!missing` to make the code easier to read and understand.

Reply:
Good point! I really like that!